nautobot/docs/release-notes/version-1.2.md

```diff
@@ -118,6 +118,24 @@ For more details, please refer to the [Secrets](../core-functionality/secrets.md
 
 Nautobot core applications and plugins can now both define panels, groups, and items to populate the Nautobot home page. The home page now dynamically reflows to accommodate available content. Plugin developers can add to existing panels or groups or define entirely new panels as needed. For more details, see [Populating the Home Page](../development/homepage.md).
 
+#### REST API Token Provisioning ([#1374](https://github.com/nautobot/nautobot/issues/1374))
```

Comment:
This should be moved to `nautobot/docs/release-notes/version-1.3.md`.

Reply:
I based off of `develop` which doesn't have that. How do you want me to approach this? Can't see the option to change the source branch of this PR. I can probably branch off of `next`, cherry-pick this on to it, and then force push my new branch over my forks `develop` branch which should update this PR.

Reply:
Ok, that's what I did basically, my forks `develop` is `nautobot/next`